src/integrations/retry.ts

```diff
@@ -142,8 +143,11 @@ export async function fetchParseBinary<T extends Message<T>>(
   parser: T,
 ): Promise<T> {
   const res = await fetch(url, fetchOpts);
+  if (isTest) {
+    // Jest only support mocks of JSON
+    return parser.fromJson(await res.json());
+  }
```

Comment:
:lolsob:

Reply:
should be able to return binary from nock by using inverse method toBinary and json object? Or does the jest not support `res.arrayBuffer`?